README.md

```diff
@@ -57,8 +57,9 @@ Ensure you have the following installed:
    using the Docker setup.
 5. **[Node](https://nodejs.org/en)** and **[`nvm` (Node Version Manager)](https://github.com/nvm-sh/nvm)** for front-end
    tooling.
-6. **[JQ](https://jqlang.github.io/jq/)** for the step in the build that installs the design system templates
-7. **Operation System**: Ubuntu/MacOS
+6. **[JQ](https://jqlang.github.io/jq/)** for the step in the build that installs the design system templates.
+7. **[MacPorts](https://www.macports.org/install.php)**: We need MacPorts so we can install the required packages to render equations using LaTeX strings via Matplotlib locally (outside of a docker container).
```

Comment:
Nit: The requirement is `texlive-latex-extra` rather than MacPorts specifically, right? Perhaps worth changing this to be more explicit

```suggestion
7. `texlive-latex-extra`: Required by `matplotlib` to render LaTeX equations. See [below](#installing-the-required-latex-packages-for-local-development) for instructions on how to install on macOS.
```